src/vma/sock/sock-redirect.cpp

```diff
@@ -762,6 +761,52 @@ int vma_dereg_mr_on_ring(int __fd, void *addr, size_t length)
 	}
 }
 
+static inline struct cmsghdr * __cmsg_nxthdr(void *__ctl, size_t __size, struct cmsghdr *__cmsg)
+{
+	struct cmsghdr * __ptr;
+
+	__ptr = (struct cmsghdr*)(((unsigned char *) __cmsg) +  CMSG_ALIGN(__cmsg->cmsg_len));
```

Comment:
Should we use C++ converters like static_cast?

Reply:
Current sock_redirect(cpp|h) source code uses `NULL`. I would prefer to keep single style inside the same module.
Once the decision has been made to move from `NULL` to `nullptr`, this can be done in a custom pull request.

Reply:
I can do this if you believe this change is required for this pull request.

Reply:
@AlexanderGrissik what is conclusion?

Reply:
Lets keep single styling